clap_complete/tests/testsuite/engine.rs

```diff
@@ -1191,3 +1191,55 @@ fn complete(cmd: &mut Command, args: impl AsRef<str>, current_dir: Option<&Path>
         .collect::<Vec<_>>()
         .join("\n")
 }
+
+#[test]
+fn suggest_value_path_with_mapper() {
```

Comment:
Please split this PR into two commits
1. Adds the test, showing the current behavior (passes)
2. Adds the feature and updates the test to use it, showing how this feature changes behavior